src/verilog.y

```diff
@@ -96,6 +96,9 @@ public:
     bool m_tracingParse = true;  // Tracing disable for parser
     bool m_insideProperty = false;  // Is inside property declaration
     bool m_typedPropertyPort = false;  // True if typed property port occurred on port lists
+    bool m_modportImpExpActive = false;  // True if the standalone ID is a tf_identifier
+                                         // instead of port_identifier
+    bool m_modportImpExpLastIsExport = false;  // True if the last import_export statement in modportPortsDecl is an export
```

Comment:
```suggestion
    bool m_modportImpExpActive = false;  // Standalone ID is a tf_identifier
                                         // instead of port_identifier
    bool m_modportImpExpLastIsExport = false;  // Last import_export statement in modportPortsDecl is an export
```